.github/workflows/awsfulltest.yml

```diff
@@ -21,12 +21,10 @@ jobs:
           compute_env: ${{ secrets.TOWER_COMPUTE_ENV }}
```

Comment:
There have been some changes in this `awsfulltest.yml` as the tests were broken. Could you change at line 17:
`nf-core/tower-action@master` to `nf-core/tower-action@v2`
Then line 20:
`bearer_token: ${{ secrets.TOWER_BEARER_TOKEN }}` to `access_token: ${{ secrets.TOWER_ACCESS_TOKEN }}`